<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Chaitanya Srinivasan</title>
  
  <meta name="author" content="Chaitanya Srinivasasn">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:10px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Chaitanya Srinivasan</name>
              </p>
              <p>I am an incoming Master's student at <a href="https://www.cmu.edu/">Carnegie Mellon University</a> <a href="https://www.cs.cmu.edu/">School of Computer Science</a> studying <a href="http://msas.cbd.cmu.edu/">Automated Science</a>. I recently completed my undergraduate also at Carnegie Mellon in <a href="http://www.cbd.cmu.edu/">Computational Biology</a>. My research advisor is <a href="http://www.pfenninglab.org/">Dr. Andreas Pfenning</a>.</p> 

              <p>Prior to beginning research with the Computational Biology Department, I interned at <a href="https://www.chp.edu/">UPMC Children's Hospital of Pittsburgh</a> in the Pediatric Gastroenterology Department. I studied the metabolomics and lipidomics of asparaginase-associated acute pancreatitis from a clinical perspective under the mentorship of <a href="https://www.husainlab.org/">Dr. Sohail Husain</a>. </p> 

              <p style="text-align:left">
                <a href="mailto:csriniv1@cs.cmu.edu">Email</a> &nbsp/&nbsp
                <a href="https://github.com/chaitanyasrinivasan">Github</a> &nbsp/&nbsp
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <img style="width:60%;max-width:90%" alt="profile photo" src="images/portrait.jpeg" class="hoverZoomLink">
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Research</heading>
              <p>
                I am broadly interested in designing computational models to study the genomic basis of neurological disorders. Currently, I am studying brain region and cell type specific epigenomics of addiction.             </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img style="width:105%;max-width:105%" src="">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="">
                <papertitle>Addiction-associated genetic variants implicate cell type- and region-specific cis-regulatory elements in addiction neurobiology (in progress) </papertitle>
              </a>
              <br>
              <b>Chaitanya Srinivasan*</b>, BaDoi N. Phan*, Alyssa J. Lawler, Easwaran Ramamurthy, Michael Kleyman, Ashley R. Brown, Irene M. Kaplow, Morgan E. Wirthlin, Andreas R. Pfenning
              <br>
              <a href="https://www.drugabuse.gov/sites/default/files/pfenning_abstract.pdf">NIDA Genetic Consortium Abstract</a>
              <br>
              <p></p>
              <p>We present a framework that maps addiction-associated genetic variation to brain region- and cell type-specific putative cis-regulatory enhancers across human and mouse epigenomic profiles. We further attribute cell-type specific heritability of addiction genetic risk using machine learning models. </p>
            </td>
          </tr>

        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img style="width:105%;max-width:105%" src="images/chaitanya_srinivasan_mom2018.png">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="images/chaitanya_srinivasan_mom2018.pdf">
                <papertitle>Using metabolomics and lipidomics to crack the code on asparaginase-associated pancreatitis (AAP) </papertitle>
              </a>
              <br>
              <b>Chaitanya Srinivasan*</b>, Li Wen, Amitava Mukherjee, Sohail Z. Husain
              <br>
              <em>Carnegie Mellon University, Meeting of the Minds Symposium, 2018.</em> 
              <br>
              <b> Won Osher Lifelong Learning Institute Award </b>
              <br>
              <a href="images/APA-Metabolomics-abstract-v2-2019-06-11.pdf">American Pancreatic Association Abstract</a>
              <p></p>
              <p>We used a non-targeted metabolomic and lipidomic platform to determine whether there are unique biomarker signatures that predict the development of asparaginase-associated pancreatitis.</p>
            </td>
          </tr>
      </td>
    </tr>

    </tbody></table>

	<table width="100%" align="center" border="0" cellspacing="0" cellpadding="20"><tbody>
	  <tr>
	    <td>
	      <heading>Projects</heading>
	    </td>  
	  </tr>
	</tbody></table>
	    <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

	    <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img style="width:105%;max-width:105%" src="images/motif_discovery.png">
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a href="https://github.com/chaitanyasrinivasan/motif_discovery">
                <papertitle>Parallel <em>de novo</em> motif discovery tool </papertitle>
              </a>
              <br>
              <b>Chaitanya Srinivasan</b>
              <br>
              <p></p>
              <p> I built tool that can perform parallel <em>de novo</em> motif discovery on regulatory genomics data. </p>
            </td>
        </tr>

	    <tr>
	        <td style="padding:20px;width:25%;vertical-align:middle">
	          <img style="width:105%;max-width:105%" src="images/02510_Poster.png">
	        </td>
	        <td style="padding:20px;width:75%;vertical-align:middle">
	          <a href="images/02510_Poster.pdf">
	            <papertitle> Linkage Optimized Ortholog Predictions (LOOP) </papertitle>
	          </a>
	          <br>
	          Noah Lee, <b>Chaitanya Srinivasan</b>
	          <br>
	          <a href="images/02510_FinalReport.pdf">Report</a>
	          <p></p>
	          <p>We devised a novel ortholog prediction method that uses simple biological assumptions and can be applied broadly.</p>
	        </td>
	    </tr>
	  <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:0px">
              <br>
              <p style="text-align:right;font-size:x-small;">
                <a style="font-size:x-small;" href="https://jonbarron.info"> Template</a>
              </p>
            </td>
          </tr>
	</tbody></table>
	</td>
	</tr>
  </table>
</body>

</html>
